content/en/docs/examples/virtual-machines/single-network/index.md

```diff
@@ -128,6 +125,27 @@ following commands on a machine with cluster admin privileges:
     {{< text bash >}}
     $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> cluster.env
     {{< /text >}}
+    
+1. Generate a `sidecar.env` to enable the cert rotation. The file contains
+   the directory path to store the cert and update the cert for cert rotation
+
+    {{< text bash >}}
+    $ echo "PROV_CERT=/var/run/secrets/istio" >> sidecar.env
```

Comment:
does something rebuild these files on a reboot? /var/run is a tmpfs and reset every reboot.